slash/log.py

```diff
@@ -200,16 +210,29 @@ def _get_file_log_handler(self, subpath, symlink, bubble=False, filter=None):
         else:
             log_path = self._normalize_path(os.path.join(root_path, _format_log_path(subpath)))
             ensure_containing_directory(log_path)
-            handler = self._get_file_handler_class()(log_path, bubble=bubble, filter=filter)
+            if use_compression:
+                log_path += ".gz" if config.root.log.compression.algorithm == "gzip" else ".brotli"
```

Comment:
I think we should only allow the two options we support, and add a failure otherwise. In the current state of things, passing `algorithm="rotem"` will pick brotli...